torch_geometric/data/temporal.py

```diff
@@ -231,12 +231,7 @@ def __cat_dim__(self, key: str, value: Any, *args, **kwargs) -> Any:
         return 0
 
     def __inc__(self, key: str, value: Any, *args, **kwargs) -> Any:
-        if 'batch' in key:
-            return int(value.max()) + 1
-        elif key in ['src', 'dst']:
-            return self.num_nodes
-        else:
-            return 0
+        return 0
```

Comment:
What's the reason for this change?

Reply:
It was like that before my first PR and I've needed to revert my change on it [here](https://github.com/pyg-team/pytorch_geometric/pull/3867/files#diff-2eace32b78b56a8f25f3204ebba7c5293d3be93d30e69267908bebc9d0853eafL113) to get it working with the DataLoader.

Reply:
Do we want to reset it?